great-idea/services.html

```diff
@@ -14,94 +14,149 @@
+          Learn More
+        </button>
+      </div>
+  </div><br></br>
+
+  <div>
+      <div class="marketing2">
+        <h5>
+          Digital Marketing
+        </h5>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button class="button2">
+          Learn More
+        </button>
+      </div>
+      <div class="webdev2">
+        <h5>
+          Web Development
+        </h5>
+        <p>
+          Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.
+        </p>
+        <button class="button2"> 
+          Learn More
+        </button>
+        </div>
+    </div><br></br>
+
+    <div>
+        <div class="iosdev2">
+          <h5>
+            iOS Development
+          </h5>
+          <p>
+            Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p>
+          <button class="button2">
+            Learn More
+          </button>
+      
+        </div>
+        <div class="anddev2">
+            <h5>
+              Android Development
+            </h5>
+            <p>
+              Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+              eleifend. Mauris euismod facilisis iaculis.
+      
+            </p>
+            <button class="button2">
+              Learn More
+            </button>
+          </div>
+      </div>
+</main><br></br>
+
+<hr>
+
+<div class="bottom2">
+  <div class="somefacts2">
+    <h5>
+    Some Facts About Our Services
+    </h5>
+    <p>
+      Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.
+    </p><br></br>
+    <ul>
+      <li>
+        Awesome thing
+      </li><br></br>
+      <li>
+        Amazing thing
+      </li><br></br>
+      <li>
+          Cool thing
+      </li><br></br>
+      <li>
+        Great thing
```

Comment:
with unordered and ordered list, items are normally put to the next lien automatically, so you dont have to add a break. if you want more spacing between it you can use line height in your css